fw/http.h

```diff
@@ -203,6 +203,7 @@ typedef enum {
 	TFW_HTTP_HDR_REGULAR,
 	TFW_HTTP_HDR_HOST = TFW_HTTP_HDR_REGULAR,
 	TFW_HTTP_HDR_CONTENT_LENGTH,
+	TFW_HTTP_HDR_CONTENT_LOCATION,
```

Comment:
Please add comment that this is response specific - with `tfw_http_msg_header_table_size()` reworking we may have further confusions with the enum and maybe we'll need to go to two enums, for requests and responses.